python/basix/ufl.py

```diff
@@ -1236,23 +1214,42 @@ def _compute_signature(element: _basix.finite_element.FiniteElement) -> str:
 
 
 @_functools.lru_cache()
-def create_element(family: _typing.Union[_basix.ElementFamily, str], cell: _typing.Union[_basix.CellType, str],
-                   degree: int, lagrange_variant: _basix.LagrangeVariant = _basix.LagrangeVariant.unset,
-                   dpc_variant: _basix.DPCVariant = _basix.DPCVariant.unset, discontinuous: bool = False,
-                   gdim: _typing.Optional[int] = None) -> BasixElement:
+def element(family: _typing.Union[_basix.ElementFamily, str], cell: _typing.Union[_basix.CellType, str], degree: int,
+            lagrange_variant: _basix.LagrangeVariant = _basix.LagrangeVariant.unset,
+            dpc_variant: _basix.DPCVariant = _basix.DPCVariant.unset, discontinuous: bool = False,
+            gdim: _typing.Optional[int] = None, rank: _typing.Optional[int] = None,
+            shape: _typing.Optional[_typing.Tuple[int, ...]] = None,
+            symmetry: _typing.Optional[bool] = None) -> BasixElement:
```

Comment:
Why does the element have to be a `BasixElement` if we are removing `ufl`? Could the element just be a `basix.ufl.FiniteElement`? Having it as a `basix.ufl.BasixElement` seems rather weird to me.

Reply:
I'm amazed that mypy was ok with this, this function can return a `BlockedElement` which is not a subclass of `BasixElement`.